### DISCUSSION TITLE
Is it possible for a template to manage its own fonts without affecting the system settings?

### DISCUSSION THREAD

**Discussion by Kangarooss** [typst/typst#4012] [Q&A]
I am currently encountering an issue with certain packages (like fontawesome0.2.0) that require downloading and installing fonts. This process significantly affects portability. For instance, whenever I use fontawesome0.2.0 on a different computer, I need to follow the package's readme to download and install the fonts manually. This is not only cumbersome but also contaminates the environment.

Is there a way for a package to bundle and carry the fonts it needs? Alternatively, is there a Typst Font Manager that can manage fonts?

**Comment by laurmaedje** [comment]
It's not currently possible, but we're aware of the problem. The best place to discuss it would be [the Discord forge thread on Resources](https://discord.com/channels/1054443721975922748/1178292717239926824). The current discussion there is about somewhat adjacent problems, but the font stuff also fits with it.
